extension/intents/music/music.toml

```diff
@@ -143,3 +143,12 @@ phrase = "play music"
 [[music.unpause.example]]
 phrase = "play spotify"
 test = true
+
+[music.find]
+ description = "Find the tab that's currently making sound"
```

Comment:
Some leading whitespace here, but also we're not finding the tab as much as we're showing the title of the tab.